BPL-CBH LCCN

```diff
@@ -0,0 +1,89 @@
+'MacroName:BPLLCCN
+'MacroDescription:LC Call version for BPL
+   
+   'LC Call number
+'Created by Miriam Gloger 
+'Copy Call number from 050 to 099 Call number for Brooklyn Public Library CBH
+
+Option Explicit
+
+Sub Main
+   Dim CS as Object
+   On Error Resume Next
+   Set CS  = GetObject(,"Connex.Client")
+   On Error GoTo 0
+   If CS  Is Nothing Then
+      Set CS  = CreateObject("Connex.Client")
+   End If
+  
+   
+  Dim sLccall as String
+  Dim sCall as String 
+  Dim sCutter as String 
+  Dim nPlace1 as Integer
+  Dim nPlace2 as Integer
+  Dim sData as String
+  Dim nBool as Integer
+  Dim z as Integer
+
+  Dim filenumber As Integer
+  Dim sFileName As String
+  Dim sDefaultInitials$
```

Comment:
It would be great to be consistent with previous syntax:
Dim sInitials$  > Dim sInitials As String
Also, I think you will need only one variable for initials. Replace sDefaultInitials with sInitials.

Reply:
done